presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveFileFormats.java

```diff
@@ -342,7 +342,7 @@ public void testOrcUseColumnNames(int rowCount)
                 .withRowsCount(rowCount)
                 .withReadColumns(Lists.reverse(TEST_COLUMNS))
                 .withSession(SESSION)
-                .isReadableByPageSource(new OrcPageSourceFactory(true, new OrcReaderOptions(), HDFS_ENVIRONMENT, STATS));
```

Comment:
orc options here should use column names

Reply:
Thanks

Reply:
Did you change anything?

Reply:
Does it mean that the test wasn't working?

Reply:
One was failing and the other one was passing.